lib/models/liturgical-day-def.ts

```diff
@@ -110,6 +110,8 @@ export default class LiturgicalDayDef implements BaseLiturgicalDayDef {
       ? input.dateExceptions
       : input.dateExceptions
       ? [input.dateExceptions]
+      : previousDef
+      ? previousDef.dateExceptions
       : [];
```

Comment:
Could this be written in a more readable way please? :wink:

```suggestion
      ? input.dateExceptions
      : input.dateExceptions
        ? [input.dateExceptions]
        : previousDef
          ? previousDef.dateExceptions
          : [];
```

Reply:
These are flattened by Prettier (we don't have hands on it). But it highlights that, generally, it's not a very good practice to chain many conditional ternary operators. I did it anyway here (as a quick fix), knowing that I already rewrote this part differently in my current refactor (for the monorepo).

Reply:
Yeah, `prettier`, I keep fighting with it. :wink:
> But it highlights that, generally, it's not a very good practice to chain many conditional ternary operators.

Yeah, but are two ternary operators considered many? 🤔

> I already rewrote this part differently in my current refactor (for the monorepo).

Okay, then disregard this conversation.

Anyway, I long to see your WIP on monorepo. Could you create a WIP/draft PR please? Thanks! :pray: